types/video.js/index.d.ts

```diff
@@ -6352,6 +6431,22 @@ export interface VideoJsPlayer extends videojs.Component {
      */
     createModal(content: string | (() => any) | Element | any[], options: any): videojs.ModalDialog;
 
+    /**
+     * Get or set the `Player`'s crossOrigin option. For the HTML5 player, this
+     * sets the `crossOrigin` property on the `<video>` tag to control the CORS
+     * behavior.
+     *
+     * @param [value]
+     *        The value to set the `Player`'s crossOrigin to. If an argument is
+     *        given, must be one of `anonymous` or `use-credentials`.
+     *
+     * @return - The current crossOrigin value of the `Player` when getting.
+     *         - undefined when setting
+     */
+    crossOrigin(value: string): void;
```

Comment:
```suggestion
    crossOrigin(value: 'anonymous' | 'use-credentials'): void;
```
>  If an argument is  given, must be one of `anonymous` or `use-credentials`.

Reply:
you can make into type (string literals union), to keep convention for those types.